src/migrations/1648079117646-bloom_backend.ts

```diff
@@ -0,0 +1,22 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class bloomBackend1648079117646 implements MigrationInterface {
+    name = 'bloomBackend1648079117646'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "user" RENAME COLUMN "languageDefault" TO "isActive"`);
```

Comment:
this doesnt seem right. it should be removing the `languageDefault` and adding the `isActive` right?